.github/workflows/integration-tests.yml

```
@@ -0,0 +1,32 @@
+name: integration-tests
```

Trying to get it to show up as `integration / tests` rather than `integration-tests / integration-tests`. Might want to change the filename to `workflows/integration.yml` too
```suggestion
name: integration
```

---

the file name doesn't really matter, the names that show up in CI are based on the [workflow name](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#name) and the [job name](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idname).